react/ActionsMenu/Actions/print.js

```diff
@@ -21,33 +21,40 @@ export const print = () => {
     icon,
     label,
     disabled: docs => docs.length === 0,
-    displayCondition: docs => isFile(docs[0]), // feature not yet supported for multi-files
+    displayCondition: docs => docs.every(doc => isFile(doc)),
     action: async (docs, { client, webviewIntent }) => {
-      const doc = docs[0] // feature not yet supported for multi-files
+      const isSingleDoc = docs.length === 1
+      const firstDoc = docs[0]
 
-      if (webviewIntent) {
-        try {
-          const blob = await fetchBlobFileById(client, doc._id)
+      try {
+        // in flagship app
+        if (webviewIntent) {
```

Comment:
nit: Why not use isFlagshipApp() from cozy-device-helper ? It is more clear than checking if webviewIntent is truthy (even if I agree it should work the same)

Reply:
j'ai pas souvenir, l'approche ici est historique, mais si tu penses qu'il vaut mieux se baser sur isFlagshipApp() pour discriminer AA / hors AA je vais regarder dans cozy-ui si on fait pas ça à plusieurs endroit...

Reply:
On utilise un partout ce isFlagshipApp en tout cas pour vérifier si on est dans l'AA (et ça a l'avantage de l'explicite et de pas avoir besoin d'un commentaire `// in flagship app`)

https://github.com/cozy/cozy-libs/blob/8119afaec991a03ce05c0082d88e4001d84a770d/packages/cozy-device-helper/src/flagship.ts#L41

> Il faut faire un if sur isFlagshipApp puis ensuite de toute façon utiliser webviewIntent, autant ne faire que le teste sur webviewIntent non ? 🤔

Je peux pas dire le contraire il faut les deux 😕 tester webviewIntent est "plus strict" que isFlagshipApp() d'une certaine manière.

Reply:
le commentaire est plus là pour séparer visuellement les blocs que pour commenter que c'est dans l'AA. Bon à voir pour plus tard :)